Proofreading - previously checked files (#11750)
openvinotoolkit/openvino: samples/python/classification_sample_async/README.md

Basic OpenVINO™ Runtime API is described in [Hello Classification Python Sample](../hello_classification/README.md).

| Options                    | Values                                                                   |
| :------------------------- | :----------------------------------------------------------------------- |
| Validated Models           | [alexnet](@ref omz_models_model_alexnet)                                 |
| Model Format               | OpenVINO Intermediate Representation (.xml + .bin), ONNX (.onnx) |
| Supported devices          | [All](../../../docs/OV_Runtime_UG/supported_plugins/Supported_Devices.md)        |
| Other language realization | [C++](../../../samples/cpp/classification_sample_async/README.md)        |

## How It Works

In the beginning, the sample application reads command-line parameters, prepares input data, loads a specified model and image(s) to OpenVINO Runtime plugin, performs synchronous inference, and processes output data, logging each step in a standard output stream.

For more information, refer to the explicit description of

> each sample **Integration Step** in the [Integrate OpenVINO Runtime with Your Application](../../../docs/OV_Runtime_UG/integrate_with_your_application.md).

Author: ```suggestion
each sample at **Integration Step** in the [Integrate OpenVINO Runtime with Your Application](../../../docs/OV_Runtime_UG/integrate_with_your_application.md).
```

Reviewer: > 

```suggestion
**Integration Steps** in the [Integrate OpenVINO Runtime with Your Application](../../../docs/OV_Runtime_UG/integrate_with_your_application.md).
```

## Running

Before running the sample, specify a model and an image:

> - You may use [public](@ref omz_models_group_public) or [Intel's](@ref omz_models_group_intel) pre-trained models from Open Model Zoo. The models can be downloaded by using the [Model Downloader](@ref omz_tools_downloader).

Author: ```suggestion
- Use [public](@ref omz_models_group_public) or [Intel's](@ref omz_models_group_intel) pre-trained models from Open Model Zoo. The models can be downloaded by using the [Model Downloader](@ref omz_tools_downloader).
```
